pkg/mcr/mcr.go

```diff
@@ -28,8 +30,10 @@ func EnsureMCRVersion(h *api.Host, specMcrVersion string) error {
 			return fmt.Errorf("%s: failed to query container runtime version after restart: %w", h, err)
 		}
 	}
-	if currentVersion != specMcrVersion {
-		return fmt.Errorf("%s: %w: container runtime version not %s after upgrade", h, constant.ErrVersionMismatch, specMcrVersion)
-	}
+
+	log.Infof("%s: mirantis container runtime version %s (requested:%s)", h, currentVersion, specMcrVersion)
```

Comment:
1. Does `mirantis` need to be capitalised? (If so, then does `container runtime` also need to be capitalised?) Genuine question; I don't want to introduce inconsistencies if it's already all-lowercase everywhere else.
2. I'm wondering if a single space should be added between `requested:` and `%s`

Reply:
```suggestion
	log.Infof("%s: Mirantis container runtime version %s (requested: %s)", h, currentVersion, specMcrVersion)
```

Reply:
happy to take both suggestions

Reply:
I'll merge after the smoke test clears. No need to restart it for such a copy change.

Reply:
I think this is better

```suggestion
	log.Infof("%s: MCR version %s (requested: %s)", h, currentVersion, specMcrVersion)
```